Emby.Server.Implementations/Data/SqliteItemRepository.cs

```diff
@@ -2957,6 +2949,7 @@ private string GetOrderByText(InternalItemsQuery query)
             {
                 var columnMap = MapOrderByField(i.Item1, query);
                 var columnAscending = i.Item2 == SortOrder.Ascending;
+                const bool enableOrderInversion = false;
```

Comment:
If this is false, can the if statement below this can be removed.

Reply:
`enableOrderInversion` was already always false, it can be removed in another PR